examples/trials/ga_squad/README.md

```diff
@@ -30,7 +32,7 @@ chmod +x ./download.sh
 ./download.sh
 ```
 
-## Download manually
+### Download manually
```

Comment:
after we add the script into command, do we still need to download manually?

Reply:
I didn't add the script to config.yml
I divided this doc into two cases: running in local/remote and running on pai. This part belong's to run the example in local, I think it's better to download the data first and run trial.py for several times.